blog/2019-05-28-first-blog-post.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This doesn't look right, this adds lots of sample Docusaurus example content

Reply:
This PR is for @j1010001  as part of the effort to integrate docusaurus to cadence-lang site.